beacon_node/lighthouse_network/src/config.rs

```diff
@@ -176,6 +176,7 @@ impl Default for Config {
             .filter_rate_limiter(filter_rate_limiter)
             .filter_max_bans_per_ip(Some(5))
             .filter_max_nodes_per_ip(Some(10))
+            .table_filter(|enr| enr.ip().map(|ip| is_global(&ip)).unwrap_or(false)) // Filter non-global IPs
```

Comment:
Slightly cleaner as:

```suggestion
            .table_filter(|enr| enr.ip().map_or(false, |ip| is_global(&ip))) // Filter non-global IPs
```

Should we also consider leaving the default as `true`? Or are peers with no IP in their ENR ineligible for connection anyway?

Reply:
Yeah, I think discovery will filter out peers without IP, but to make sure we also don't want any peers without IPs in our discovery table.